src/Kaleidoscope.h

```diff
@@ -34,6 +34,11 @@ extern HARDWARE_IMPLEMENTATION KeyboardHardware;
 
 #define KEYMAP_SIZE (sizeof(keymaps) / ROWS / COLS / sizeof(Key))
 
+#define USE(plugins...) ({                                      \
```

Comment:
The hack is brilliant. But the macro name may be too generic, since it's not namespaced. USE_PLUGINS? 

Also, I'd love a few lines of doc in the code about what it's doing and why. 

Generally, the more magic the code, the more we need doc explaining our intent.